network/vpc/api-cli/vpc-private-beta.mdx

```diff
@@ -0,0 +1,101 @@
+---
+meta:
+  title: VPC Regional Private Networks - Private Beta Guide
+  description: This page shows you how to get started with Scaleway VPC Regional Private Networks, if you are one of the Private Beta testers.
+content:
+  h1: VPC Regional Private Networks - Private Beta Guide
+  paragraph: This page shows you how to get started with Scaleway VPC Regional Private Networks, if you are one of the Private Beta testers.
+dates:
+  validation: 2022-02-14
+  posted: 2021-02-14
+categories:
+  - network
+---
+
+VPC Regional Private Networks allow you to create Private Networks that are scoped to a region (Paris, Amsterdam or Warsaw), rather than a specific [Availability Zone](/compute/instances/concepts/#availability-zone) (AZ). This means you can attach resources from multiple AZs of that region, to your Private Network.
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You are part of the VPC Regional Private Networks Private Beta
+  - You have [created an API key](/console/my-project/how-to/generate-api-key/)
+</Message>
+
+## Getting started
+
+1. [Create a new Project](/console/my-project/how-to/create-a-project/). You may use an existing Project, but this is at your own risk. We recommend creating a new Project for the purposes of testing this beta product.
+
+2. Email the Project ID to <Lightbox src="scaleway-email-add.webp" size="large" />
+
```

Comment:
This whole section 2 can be removed now that we have the migration procedure.
People can either create a project and add regional PNs using the API or else migrate their own existing resources